.eslintrc.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
         'eslint:recommended',
         'plugin:@typescript-eslint/recommended',
         'plugin:@typescript-eslint/recommended-requiring-type-checking',
-        'plugin:vue/essential',
+        'plugin:vue/vue3-essential',
```

Comment:
*(Arbitary comment location)*

On the send page, if there is an error in the "recipient" field, and the user tries to edit the "amount" field, the "recipient" field steals the cursor focus every time a change is made to the "amount" field.

![recipient-error-steals-focus](https://user-images.githubusercontent.com/1479802/213532514-a2c5b387-b7be-4f49-9bec-c66dcc31e097.gif)

Reply:
Ha nice find. Here's why it happens:
- We have a watcher to watch for changes in a few form-related parameters
- One is the amount. When amount changes, we validate the new amount AND validate the full form
- So when you type the first character in the amount, the form gets validated. The recipient is validated, and validation fails, resulting in the focus on the send component

Interestingly, it seems this doesn't exist on master, so I'm guessing I introduced it in 9e88061 when modifying that watcher to fix payment link parsing. This isn't _terrible_ since you'll need to fix the recipient anyway, and I think this whole watcher setup we have should be cleaned up and refactored since a lot of it was to make sure things worked properly when we didn't refresh on network change, so I pulled this out in https://github.com/ScopeLift/umbra-protocol/issues/441 to track.

Still open to getting the fix in during this PR, lmk what you think

Reply:
If there's an easy fix (even a hacky one) I think it would be better to fix it if possible. Obviously it's poor UX, but also there's a risk it could lead to a user mangling the recipient address, changing one character by accident or something because they don't know what's going on, and sending funds to the wrong place. Relatively low probability but still. That said if it's a big pain to fix, then I'm ok merging to get this PR in, as long as we can come back to this relatively quickly to get it fixed.

Reply:
Ok this _should_ be good now. Tested it a bit and it seems to work. Needs some more testing, both in normal flow (visit send page and use it) and payment link flow (load a payment link and maybe modify it). Also pushed the banner changes